integrations/django.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title: Django integration
+title_nav: Django
+description: Django TinyMCE component.
+keywords: integration integrate Django django django-tinymce python
+---
+
+{{site.thirdPartyInteg}}
+
+For information on adding {{site.productname}} to a Django project, try using the Python-based `django-tinymce` package. For information on setting up `django-tinymce`, see: [the django-tinymce Documentation](https://django-tinymce.readthedocs.io/en/latest/index.html).
+To use an newer version of {{site.productname}}, use the {{site.cloudname}} or statically host a manually downloaded copy of {{site.productname}}.
+
+## Using the Tiny Cloud with django-tinymce
+
+To load {{site.productname}} from the {{site.cloudname}}, update the project’s `settings.py` file:
+
+* Set `TINYMCE_JS_URL` to the {{site.cloudname}} CDN.
+* Ensure `TINYMCE_COMPRESSOR` is set to `False`.
+
+For example:
+
+```py
+TINYMCE_JS_URL = '{{site.cdnurl}}'
+TINYMCE_COMPRESSOR = False
+```
+
+## Using a manually downloaded instance TinyMCE with django-tinymce
```

Comment:
There's an `of` missing here and should this say self-hosted instead of manually downloaded?